src/race.c

```diff
@@ -271,6 +271,23 @@ void ToggleRace(void)
 	apply_race_settings();
 }
 
+void RaceCountdownChange(float t)
+{
+	if (match_in_progress || !isRACE() || race_is_started())
+        {
+		return;
+	}
+
+	float rcd = cvar("k_race_countdown") + t;
```

Comment:
Could you move this to the beginning of the function?
The code style is using the traditional C syntax, meaning that variable definitions/declarations are at the top of a block.